lib/resources/r4/account.yaml

```diff
@@ -0,0 +1,69 @@
+---
+name: Account
+field_name_base_url: http://hl7.org/fhir/r4/account-definitions.html#Account
+fields:
+- name: identifier
+  required: 'Yes'
+  cardinality: 0..*
+  type: Identifier
+  description: A business identifier of the account.
+  action: terminology
+  url: https://hl7.org/fhir/R4/account-definitions.html#Account.identifier
+  binding:
+    description: A business identifier of the account.
+    terminology:
+    - display: PFT Bill Alias Type
+      system: https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/28200
+      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes/#code-set-28200-pft-bill-alias-type
+    - display: Account Number
+      system: https://fhir.cerner.com/accountnumber
+      info_link: .
+
+- name: type
+  required: 'Yes'
```

Comment:
This field is not required

Reply:
https://github.com/cerner/fhir.cerner.com/commit/62764511c7f719faa5141a6b7bcbfcc839226e9f#diff-5a59138a5bd3eaa9ab30d592af99e473R23

+1